future-of-lightning/ANSWERS.md

```diff
@@ -1,9 +1,9 @@
 # Some Responses to the Discussion Questions
 
 ### eltoo
-Q: why do we need sighash_noinput for eltoo?  
+Q: Why do we need SIGHASH_NOINPUT for eltoo?  
 A: Instead of having to publish the entire sequence of update transactions all the way to your most recent settlement transaction, SIGHASH_NOINPUT allows you to skip over all update transactions until the one you need.
 
 ### Submarine swaps
-Q: Can an adversary make a swap and wait for the last minute before the lock expires and then reveal the secret on one chain and tries to use the signature on the other? If the blocks there are pretty fast he might be able to get off with the money from both chains.  
+Q: Can an adversary make a swap and then wait until the last moment before the lock expiry before revealing the secret on one chain and using the signature on the other? If the blocks are fast, could an attacker be able to steal the money from both chains?  
```

Comment:
Does line 8 read correctly here? "then wait until the last moment before the lock expiry before revealing."